pallets/moonbeam-xcm-benchmarks/src/weights/generic.rs

```diff
@@ -395,6 +396,16 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 		// Minimum execution time: 500_000_000_000 picoseconds.
 		Weight::from_parts(500_000_000_000, 0)
 	}
+	/// Storage: Benchmark Override (r:0 w:0)
+	/// Proof Skipped: Benchmark Override (max_values: None, max_size: None, mode: Measured)
+	/// The range of component `x` is `[1, 1000]`.
+	fn export_message() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `0`
+		//  Estimated: `0`
+		// Minimum execution time: 18_446_744_073_709_551_000 picoseconds.
+		Weight::from_parts(500_000_000_000, 0)
```

Comment:
I'm guessing `500_000_000_000` was hard coded, but why the huge `Minimum execution time` number?